### ISSUE TITLE
Please update globby to latest (8.0.2)

### ISSUE THREAD

**Issue by peterjwest** [zeit/pkg#603]
(8.0.2 is a bugfix for a breaking dir-glob change)

**Comment by randallknutson** [comment]
Our CI system was completely down because of this. I've added a pull request to update it. In the meantime, anyone who needs an emergency build can install from: 
```
npm install -g https://github.com/randallknutson/pkg.git#build
```

**Comment by Simplonium** [comment]
Yep, we have to update globby manually now.

**Comment by igorklopov** [comment]
Fixed in `pkg@4.3.6`
